pkg/daemon/ceph/agent/flexvolume/server.go

```diff
@@ -271,7 +271,12 @@ func copyFile(src, dest string) error {
 	if err != nil {
 		return errors.Wrapf(err, "error copying file from %s to %s", src, dest)
 	}
-	return destFile.Sync()
+	sync := destFile.Sync()
```

Comment:
You can still name it err since the err declared in the `if` statement is in a different scope, so line 278 will then return the original err as desired.
```suggestion
	err := destFile.Sync()
	if err := destFile.Close(); err != nil {
		return err
	}
	return err

```